File: testsuite/communication_file_io/tests/python_ac/ac.py
import sys
sys.stdin = open('cipher.inp', 'r')
sys.stdout = open('cipher.out', 'w')

import time
time.sleep(1)

command, data = input().split(' ')
if command == "ENCODE":
    print("lets_pretend_this_is_a_ciphertext_" + data)
else:
    print(data[34:])

File: testsuite/communication_file_io/tests/cpp_ac/ac.cpp
#include <iostream>

using namespace std;

int main() {
  freopen("cipher.inp", "r", stdin);
  freopen("cipher.out", "w", stdout);
  string command, data;
  cin >> command >> data;
  if (command == "ENCODE") {
    cout << "lets_pretend_this_is_a_ciphertext_" << data << endl;
  } else {
    cout << data.substr(34) << endl;
  }
  return 0;
}
